tasks/alerting.py

```diff
@@ -61,17 +67,62 @@ def send_alert_discord(context):
                 },
                 {
                   "name": "Date",
-                  "value": execution_date,
+                  "value": str(context.get('logical_date')),
                   "inline": True
                 }
-          ]
-        }
-      ]
-    };
+              ]
+            }
+        ]
+    }
+
     webhook = HttpHook(http_conn_id="discord")
-    response = webhook.run(
-            endpoint=Variable.get("DISCORD_WEBHOOK"),
-            headers={"Content-type":"application/json"},
+    responses = []
+    
+    for webhook_url in webhooks:
+        response = webhook.run(
+            endpoint=webhook_url,
+            headers={"Content-type": "application/json"},
             data=json.dumps(payload)
         )
-    return response
+        responses.append(response)
+    
+    return responses[0] if len(responses) == 1 else responses
+
+# Alert for dbt tests failing
+def send_alert_discord_dbt_tests(context):
+    task_instance = context.get('task_instance')
+    log_url = task_instance.log_url
+    
+    try:
+        log_content = requests.get(log_url).text
+        match = re.search(r'Failure in test (.*?)[\s]+', log_content)
+        error_message = match.group(0).strip() if match else "DBT test failure message not found."
+    except Exception as e:
+        error_message = f"Error extracting failure message: {e}"
+
+    model_name = get_model_name_from_task(task_instance)
+    webhooks = get_webhooks_for_model(model_name)
+
+    return send_discord_msg(context, webhooks, "DBT failure", error_message)
```

Comment:
```suggestion
    return send_discord_msg(context, webhooks, "Tranformation Test Failure", error_message)
```

The transformation hasn't failed, only the test